tests/test_downloaders.py

```diff
@@ -85,7 +84,7 @@ def setUp(self):
         self.response.headers['Content-Disposition'] = f'inline;filename="{self.file_name}"'
         self.response.headers['Content-Length'] = 1
         self.response.status_code = 200
-        self.response._content = bytes(self.response_text, 'utf-8')  #pylint:disable=protected-access
```

Comment:
I don't think that type of comment is useful, thus I remove that comment. I will return it back although it is useless.

Reply:
It's not useless, it alters pylint's behavior so that it does not signal an error.
In that case, we need to access a private attribute for testing purposes, so we disable the corresponding pylint rule.
It avoids being swamped with useless messages.

Reply:
Done in the next commit